src/docs/06-docs-as-code-course.mdx

```diff
@@ -0,0 +1,98 @@
+* **Lesson 2: Topic types (+ downloadable templates)** — Explore different types of documentation topics and utilize our specially designed templates to streamline your writing process.
+* **Lesson 3: Style Guide** — Learn how to maintain consistency and professionalism in your content by adhering to a well-defined style guide.
+
+**Webinar: Q&A about content production in a Docs as Code workflow** — Join us for an interactive Q&A session about content production in a Docs as Code workflow, where you can get insights and answers from our course instructor and guest experts.
+
+## Module 3: Reviewing your content
+
+This module focuses on enhancing the quality and effectiveness of your documentation through thorough review processes.
+
+* **Lesson 1: Docs as Code review process** — Dive into the specific review processes unique to Docs as Code, ensuring accuracy and quality in your documentation. 
+* **Lesson 2: Automation, linters** — Learn about the tools and techniques for automating your documentation processes, including the use of linters to maintain content quality.
+* **Lesson 3: Editing, collaboration, contribution** — Explore best practices in editing, the dynamics of collaborative writing, and strategies for encouraging contributions. 
+
+**Webinar:** Engage in a lively Q&A session about the nuances of reviews, collaboration, and contribution within a Docs as Code workflow, featuring insights from our course instructor and industry experts.
+
+## Module 4: Publishing
+
+This module delves into the final stages of the Docs as Code process, focusing on preparing and deploying your documentation.
+
+* **Lesson 1: Environments** — Understand the different environments in Docs as Code, such as development, staging, and production, and learn how to manage your documentation across these. 
+* **Lesson 2: Merging and deploying** — Gain insights into the best practices for merging changes and deploying updated content, ensuring a smooth and efficient publication process. 
+
+**Webinar:** Participate in an informative Q&A session centered around the challenges and best practices of publishing in a Docs as Code environment. This interactive webinar, led by our course instructor and guest experts, offers an opportunity to deepen your understanding and address specific queries.
+
+## Conclusion
+
+As we wrap up the Docs as Code course, this concluding video will guide you through the next steps in your documentation journey. We'll share a curated list of advanced learning resources and tools to further enhance your skills. Reflecting on what you've learned, we'll summarize the key takeaways from the course and how they apply to real-world scenarios. This video is designed to leave you equipped, inspired, and ready to apply your newfound knowledge in your professional endeavors.
+
+
+# Brought to you by
+
+## Instructor
+
+<Grid>
```

Comment:
We can try something like this:


```mdx
  <Grid cols='3'>
    ![Photo of Adam Broadway](/adam_broadway.png)
    <div style={{ gridColumn: 'span 2 / span 2' }}>
      <Card>
      **Adam Broadway**  
      *Founder and CEO, platformOS*  
    
      Adam has always prioritized high-quality documentation, recognizing its crucial role in [platformOS](https://www.platformos.com/)'s success. Under his leadership, the platform's documentation became a key asset, evident in the success of the award-winning platformOS Developer Portal. Adam's advocacy for the Docs as Code methodology stems from its proven impact on improving documentation quality, simplifying contribution, and building a strong community. With this course, he aims to share the transformative power of Docs as Code, demonstrating how it can drive success across various fields.
      </Card>
    </div>
  </Grid>
```